docs/deep_linking.md

```diff
@@ -0,0 +1,87 @@
+---
+id: deep_linking
+title: Deep Linking
+sidebar_label: Deep Linking
+---
+All Kiln Nav Drawers, i.e. the ui elements that slide in from the right and left, can be deep-linked to by manipulating the url hash. All page components and their property modals can also be deep-linked to as the url hash is updated when they are being edited to reflect the name of the component.
```

Comment:
let's change `ui` to `UI` because is an acronym

Reply:
I think it would be better if we make the suggestion so you know where to update the docs. Here it is
```suggestion
All Kiln Nav Drawers, i.e. the UI elements that slide in from the right and left, can be deep-linked to by manipulating the url hash. All page components and their property modals can also be deep-linked to as the url hash is updated when they are being edited to reflect the name of the component.
```